moncli/entities/column_value/simple.py

```diff
@@ -1,15 +1,77 @@
 from moncli.error import ColumnValueError
 from moncli.entities.column_value.objects import PersonOrTeam
 from moncli import enums
-
+from datetime import datetime, timedelta
 from .base import SimpleNullValue, ComplexNullValue
 from .constants import SIMPLE_NULL_VALUE
 from ...error import ColumnValueError
+from moncli.config import DATE_FORMAT,TIME_FORMAT
+import pytz
 
 class DateValue(ComplexNullValue):
     """A date column value."""
-    pass
+    native_type = datetime
+    allow_casts = (int, str)
+    has_time = False
+
+    def _convert(self,value):
+        try:
+            new_time = datetime.strptime(value['time'], TIME_FORMAT)
+            new_date = datetime.strptime(value['date'], DATE_FORMAT)
+            self.has_time = True
+            new_date = pytz.timezone('UTC').localize(new_date)
+            new_date = new_date + timedelta(hours=new_time.hour, minutes=new_time.minute, seconds=new_time.second)
+            date_value = new_date.astimezone(datetime.now().astimezone().tzinfo) 
+            return date_value
+
+        except KeyError:
+            new_date = datetime.strptime(value['date'], DATE_FORMAT)
+            return new_date
         
+    def _cast(self,value):
+        if isinstance(value,int):
+            try: 
+                new_date_value =  datetime.fromtimestamp(value)
+                date=str(new_date_value.date())
+                time=str(new_date_value.time())
+                date_dict = dict(date=date,time=time)
+                return self._convert(date_dict)
+
+            except ValueError:
+                raise ColumnValueError(
+                    'invalid_unix_date',
+                     self.id,
+                    'Unable to convert "{}" from UNIX timestamp.'.format(value)
+                )
+
+        if isinstance(value, str):
+            date_value = value.split()
+            return_value = {}
+            try:
+                if len(date_value) == 1:
+                    return_value['date'] = value
+                    return self._convert(return_value)
+                if len(date_value) == 2:
+                    return_value['date'] = date_value[0]
+                    return_value['time'] = date_value[1]
+                    return self._convert(return_value)
```

Comment:
Same thing here.  Luckily there is a simpler way.  In the case to where you have both a date and time (assumed local), you can create a new format like '{} {}'.format(DATE_FORMAT, TIME_FORMAT) to concatenate both formats with a space in the middle and return a datetime.strptime(value, your_new_format).  Notice that I am passing in the raw string value and not the date_value split.